src/test/java/fr/inria/coming/spoon/repairability/RepairabilityTest.java

```diff
@@ -24,6 +27,18 @@ public void testRepairabilityInterface() throws Exception {
         TestUtills.runRepairability("JMutRepair", "/repairability_test_files/one/");
     }
 
+    @Test
+    public void testDiffResults() throws Exception {
+        FinalResult result = TestUtills.runRepairabilityPrint("ALL", "repogit4testv0");
```

Comment:
not necessary, ideally you can directly call runRepairability

Reply:
runRepairability is a static method in a class TestUtills, do you want me to omit the class? @monperrus